src/plugin/indexer-solr/src/java/org/apache/nutch/indexwriter/solr/SolrConstants.java

```diff
@@ -19,19 +19,23 @@
 public interface SolrConstants {
   public static final String SOLR_PREFIX = "solr.";
```

Comment:
Any reason to remove all of these as well?

Reply:
It's still used for {{ZOOKEEPER_HOSTS = SOLR_PREFIX + "zookeeper.hosts"}}